src/components/SearchPage.vue

```diff
@@ -169,7 +169,7 @@
 
 
     <div class="p-1">
-      <b-container fluid>
+      <b-container fluid :class="{'opacity-50': $store.state.processing_status}">
```

Comment:
I would remove this feature entirely as for bad connections the transparency stays more than the loading info itself, making just one portion of the search page half transparent.
Let's make it simpler ;)